.travis.yml

```diff
@@ -0,0 +1,57 @@
+deploy:
+  provider: releases
+  api_key:
+      secure:
+  file:
+      - dist/makerlog-menubar-mac-*.zip
+      - dist/makerlog-menubar-mac-*.dmg
+      - dist/makerlog-menubar-windows-*.exe
+      - dist/makerlog-menubar-linux-*.tar.gz
+      - dist/makerlog-menubar-linux-*.snap
+      - dist/makerlog-menubar-linux-*.deb
+      - dist/makerlog-menubar-linux-*.rpm
+  file_glob: true
+  skip_cleanup: true
+  on:
+    repo: digitalsparky/makerlog-menubar
```

Comment:
```suggestion
    repo: Booligoosh/makerlog-menubar
```

Reply:
Oops, thanks for that, 3am pull request fun lol